internal/quickstart/container.go

```diff
@@ -46,6 +56,7 @@ type ContainerModel struct {
 }
 
 func NewContainerModel(
+	analyticsTracker analytics.Tracker,
```

Comment:
It looks like we don't set this on the container model, so it panics later when calling a method on it.